.github/actions/publish-image/action.yml

```diff
@@ -30,11 +30,11 @@ runs:
       shell: bash
       run: |
         if [[ "${{ inputs.branch_name }}" == "master" ]]; then
-            TAG_VERSION=$(grep -m1 version package.json | awk '{ print $2 }' | sed 's/[", ]//g')
+            TAG_VERSION=$(jq -cr '.version' < package.json)
         else
             TAG_VERSION=${{ inputs.branch_name }}-${{ github.run_id }}
         fi
         echo "TAG_VERSION=$TAG_VERSION" >> $GITHUB_ENV
     - name: Publish image
       shell: bash
-      run: ./scripts/travis/deploy/publish.sh ${{ inputs.domain }} ${{ inputs.username }} ${{ inputs.password }} $TAG_VERSION ${{ inputs.branch_name }} ${{ inputs.dry-run }}
+      run: ./scripts/travis/deploy/publish.sh "${{ inputs.domain }}" "$REPO_SLUG" "${{ inputs.username }}" "${{ inputs.password }}" "$TAG_VERSION" "${{ inputs.branch_name }}" "${{ inputs.dry-run }}"
```

Comment:
strictly speaking the inputs variable don't need to be double quoted as those are not bash variables (unless you want to make sure they are read by your command as a single argument). Though that's OK

Reply:
thanks, I'm being on the safe side here